saleor/core/jwt_manager.py

```diff
@@ -170,6 +177,10 @@ def validate_configuration(cls):
         except Exception as e:
             raise ImproperlyConfigured(f"Unable to load provided PEM private key. {e}")
 
+    @classmethod
+    def get_issuer(cls) -> str:
+        return build_absolute_uri(reverse("api"))
```

Comment:
Could it be possible to add this change so the domain can be easily overridden?
```suggestion
    @classmethod
    def get_issuer(cls) -> str:
        return build_absolute_uri(reverse("api"), domain=cls.get_domain())
```